application/src/js/cms/_scroll_past_sticky_status_banner.js

```diff
@@ -0,0 +1,28 @@
+;(
+    function () {
+        var errorBox = document.querySelector('.flash-message .alert-box.error');
+
+        if (!isUndefinedOrNull(errorBox)) {
+            var errors = errorBox.querySelectorAll('a')
+
+            var scrollForStickyHeader = function (event) {
+                window.setTimeout(function () {
+                    var href = this.getAttribute('href');
+                    var targetElement = document.querySelector(href);
+
+                    targetElement.scrollIntoView();
+
+                    var statusBanner = document.querySelector('.status.sticky-js');
+                    if (!isUndefinedOrNull(statusBanner)) {
+                        var statusBannerHeight = $('.status.sticky-js').outerHeight(true);
```

Comment:
I think we should try and stick to vanilla javascript, and use `scrollHeight` here? Might need restructuring the sticky banner to avoid the use of margin though.